metric_learn/_util.py

```diff
@@ -324,7 +326,32 @@ def check_collapsed_pairs(pairs):
                        "in total.".format(num_ident, pairs.shape[0]))
 
 
-def transformer_from_metric(metric):
+def _check_sdp_from_eigen(w, tol=None):
+  """Checks if some of the eigenvalues given are negative, up to a tolerance
+  level, with a default value of the tolerance depending on the eigenvalues.
+
+  Parameters
+  ----------
+  w : array-like, shape=(n_eigenvalues,)
+    Eigenvalues to check for non semidefinite positiveness.
+
+  tol : float, optional
+    Negative eigenvalues above - tol are considered zero. If
+    tol is None, and w are `metric`'s eigenvalues, and eps is the

```

Comment:
"and w are `metric`'s eigenvalues" is not needed

Reply:
That's right, done